db/migrate/20161118235506_add_missing_indexes_across_several_tables.rb

```diff
@@ -0,0 +1,26 @@
+class AddMissingIndexesAcrossSeveralTables < ActiveRecord::Migration
+  def change
```

Comment:
- Assignment Branch Condition size for change is too high. [22/20]
- Method has too many lines. [22/20]